src/mlpack/tests/bayesian_linear_regression_test.cpp

```diff
@@ -115,13 +116,13 @@ BOOST_AUTO_TEST_CASE(OptionsMakeModelDifferent)
   blrCS.Train(matX, y);
 
   for (size_t i = 0; i < nDims; ++i)
-    BOOST_REQUIRE((blr.Omega()(i) != blrC.Omega()(i)) &&
-                  (blr.Omega()(i) != blrCS.Omega()(i)) &&
-                  (blrC.Omega()(i) != blrCS.Omega()(i)));
+    REQUIRE(((blr.Omega()(i) != blrC.Omega()(i)) &&
+             (blr.Omega()(i) != blrCS.Omega()(i)) &&
+             (blrC.Omega()(i) != blrCS.Omega()(i))));
```

Comment:
We could actually split these into separate conditions, which might look a bit nicer.

```suggestion
    REQUIRE(blr.Omega()(i) != blrC.Omega()(i));
    REQUIRE(blr.Omega()(i) != blrCS.Omega()(i));
    REQUIRE(blrC.Omega()(i) != blrCS.Omega()(i));
```
I think I got everything right in that suggestion. :)